Tools/WinMLRunner/src/Run.cpp

```diff
@@ -293,13 +293,6 @@ HRESULT CheckIfModelAndConfigurationsAreSupported(LearningModel& model, const st
         else if (inputFeature.Kind() == LearningModelFeatureKind::Tensor)
         {
             auto tensorFeatureDescriptor = inputFeature.try_as<TensorFeatureDescriptor>();
-            if (tensorFeatureDescriptor.Shape().Size() > 4 && deviceType != DeviceType::CPU)
```

Comment:
should you leave this check and update the check and message to allow up to 5d tensors and not 6?

Reply:
I'm adding 6D, 7D, 8D in just a few weeks. Please don't put artificial limitations in a tool, when the API should check it.